<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForgotPasswordDialog</class>
 <widget class="QDialog" name="ForgotPasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HospAI - 忘记密码</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string>QDialog {
    background-color: #f8f9fa;
}

QFrame#resetContainer {
    background: white;
    border: 2px solid #dee2e6;
    border-radius: 15px;
    padding: 30px;
}

QLabel#logoLabel {
    font-size: 80px;
    color: #2c3e50;
    margin: 10px 0 15px 0;
    min-height: 90px;
}

QLabel#titleLabel {
    font-size: 32px;
    font-weight: bold;
    color: #2c3e50;
    margin: 10px 0 15px 0;
    min-height: 40px;
}

QLabel#subtitleLabel {
    font-size: 16px;
    color: #6c757d;
    margin: 10px 0 20px 0;
    line-height: 1.6;
    min-height: 50px;
}

QLabel#formLabel {
    font-size: 16px;
    font-weight: bold;
    color: #495057;
    margin: 15px 0 8px 0;
}

QLabel#usernameLabel {
    font-size: 16px;
    font-weight: bold;
    color: #495057;
    margin: 15px 0 8px 0;
}

QLabel#emailLabel {
    font-size: 16px;
    font-weight: bold;
    color: #495057;
    margin: 25px 0 8px 0;
}

QLineEdit {
    padding: 18px 20px;
    border: 2px solid #E5E5EA;
    border-radius: 12px;
    font-size: 16px;
    background-color: white;
    min-height: 25px;
    margin: 0 0 20px 0;
}

QLineEdit:focus {
    outline: none;
    border-color: #007AFF;
    background-color: #ffffff;
}

QLineEdit:hover {
    border-color: #B3B3BB;
}

QPushButton#resetBtn {
    width: 100%;
    background-color: #007AFF;
    color: white;
    border: none;
    border-radius: 12px;
    padding: 20px;
    font-size: 18px;
    font-weight: bold;
    margin: 25px 0 15px 0;
    min-height: 25px;
}

QPushButton#resetBtn:hover:enabled {
    background-color: #0056CC;
}

QPushButton#resetBtn:pressed:enabled {
    background-color: #004085;
}

QPushButton#resetBtn:disabled {
    background-color: #C7C7CC;
    color: #ffffff;
}

QPushButton#backLink {
    background: transparent;
    border: none;
    color: #007AFF;
    font-size: 16px;
    text-decoration: underline;
    padding: 12px;
    margin: 15px 0;
}

QPushButton#backLink:hover {
    color: #0056CC;
}

QLabel#messageLabel {
    padding: 15px 20px;
    border-radius: 10px;
    font-size: 16px;
    margin: 20px 0;
    min-height: 20px;
}

QLabel#messageLabel[type="success"] {
    background-color: #e8f5e8;
    color: #2e7d32;
    border: 1px solid #c8e6c9;
}

QLabel#messageLabel[type="error"] {
    background-color: #ffebee;
    color: #c62828;
    border: 1px solid #ffcdd2;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>40</number>
   </property>
   <property name="topMargin">
    <number>40</number>
   </property>
   <property name="rightMargin">
    <number>40</number>
   </property>
   <property name="bottomMargin">
    <number>40</number>
   </property>
   <item>
    <spacer name="topSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="resetContainer">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="containerLayout">
      <property name="spacing">
       <number>30</number>
      </property>
      <property name="leftMargin">
       <number>50</number>
      </property>
      <property name="topMargin">
       <number>40</number>
      </property>
      <property name="rightMargin">
       <number>50</number>
      </property>
      <property name="bottomMargin">
       <number>40</number>
      </property>
      <item>
       <widget class="QLabel" name="logoLabel">
        <property name="text">
         <string>🔒</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>找回密码</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="subtitleLabel">
        <property name="text">
         <string>请输入注册时的用户名和邮箱，我们将发送重置密码链接</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="headerSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>25</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="formLayout">
        <property name="spacing">
         <number>30</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="usernameLayout">
          <property name="spacing">
           <number>12</number>
          </property>
          <item>
           <widget class="QLabel" name="usernameLabel">
            <property name="text">
             <string>用户名</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="usernameEdit">
            <property name="placeholderText">
             <string>请输入用户名</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="emailLayout">
          <property name="spacing">
           <number>12</number>
          </property>
          <item>
           <widget class="QLabel" name="emailLabel">
            <property name="text">
             <string>邮箱</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="emailEdit">
            <property name="placeholderText">
             <string>请输入绑定邮箱</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="formSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>30</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="resetBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>发送重置链接</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="messageLabel">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="messageSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="backLink">
        <property name="text">
         <string>← 返回登录</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="bottomSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 